src/sentry/notifications/activity/unassigned.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any, Mapping, Tuple
+
+from .base import ActivityNotification
+
+
+class UnassignedActivityNotification(ActivityNotification):
+    def get_activity_name(self) -> str:
+        return "Unassigned"
+
+    def get_description(self) -> Tuple[str, Mapping[str, Any], Mapping[str, Any]]:
+        return "{author} unassigned {an issue}", {}, {}
```

Comment:
can we use an f string here?

Reply:
This string is passed as `description` to: 
```
return description.format(**context)
```
so it needs to remain is this weird format.